app/controllers/concerns/projects_params.rb

```diff
@@ -0,0 +1,11 @@
+module ProjectsParams
+  def resource_params
+    params.require(:project).try(
```

Comment:
Great suggestion!

I thought of something similar, but I wasn't sure it worked, so I opted for the `#try` version.

Note: this solution works only from Rails 4, where `#try` behavior has been updated to swallow `NoMethodErrors` even if the `self` object is not `nil`.